test/js/weak2-test.js

```diff
@@ -22,10 +22,10 @@ test('weak2', function (t) {
   });
 
   // run weak callback, should dispose
-  gc();
+  require('./gc-fn')();
 
   // do not run weak callback
-  gc();
+  require('./gc-fn')();
```

Comment:
Not a blocker but I would like it if you `, gc = require('./gc-fn')` at the top of the file.

(Also applies to the other tests, of course.)